### DISCUSSION TITLE
PiPedal v1.3.69 Release

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#256] [General]
#### Important notice: PiPedal has a submitted a request to support sharing of Pipedal presets at [PatchStorage.com](https:///patchstorage.com). We urgently need your support! Please vote for PiPedal [here](https://patchstorage.com/requests/pipedal/).

Bug fixes:
- Fix file uploads for TooB NAM and TooB Convolution Reverb.
- Load actual list of Wi-Fi regulatory domains from regulatory database.
- Reload channel list after selecting a new Wi-Fi regulatory domain.
- Hotspot configuration dialog: validate country code and channel selection.
- Add `--fix-permissions` and `-list-wifi-country-codes options` to `pipedalconfig`
- Correct error in `wifi_latency_test` help text.
- Cleaned up Debian package dependencies.
- Correct bug with mDNS service announcements after a name collision.
- Cleaned up build prerequisite documentation for PiPedal and ToobAmp projects.
- Clean service shutdown: avoid double-close of web server sockets.
- Numerous fixes for Ubuntu 24.04 LTS builds for arm64 and amd64. (experimental)

 
A complete set of release notes can be found [here](https://rerdavies.github.io/pipedal/ReleaseNotes.html).

---
The .asc file is a gpg digital signature for the .deb file. You can verify the signature using the following command:

    gpg --verify pipedal_1.3.69_arm64.deb.asc pipedal_1.3.69_arm64.deb

Files are signed with the following key:

&nbsp;&nbsp;&nbsp;&nbsp;[PiPedal Project &lt;rerdavies@gmail.com&gt; - 2D1F 39DB B1F8 1941 2B67  8F88 E9D7 081E 08E3 D85C ](https://rerdavies.github.io/pipedal/pipedal-public-key.asc)

<hr /><em>This discussion was created from the release <a href='https://github.com/rerdavies/pipedal/releases/tag/v1.3.69'>PiPedal v1.3.69 Release</a>.</em>

**Comment by BorisSutin** [comment]
Thanks Robin!
